.github/workflows/check-status.yml

```diff
@@ -47,7 +47,7 @@ jobs:
               const result = await github.rest.checks.listSuitesForRef({
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                ref: "${{ github.head_ref }}"
+                ref: "${{ github.event.pull_request.head.sha }}"
```

Comment:
wouldn t this cause an issue if the run is a workflow_dispatch, as event.pull_request will not exist ?